src/System.Management.Automation/engine/hostifaces/PowerShell.cs

```diff
@@ -860,6 +861,25 @@ public static PowerShell Create(InitialSessionState initialSessionState)
             return result;
         }
 
+        /// <summary>
+        /// Constructs an empty PowerShell instance; a script or command must be added before invoking this instance.
+        /// </summary>
+        /// <param name="runspace">Runspace in which to invoke commands.</param>
+        /// <returns>An instance of PowerShell.</returns>
+        public static PowerShell Create(Runspace runspace)
+        {
+            if (runspace == null)
+            {
+                throw new PSArgumentNullException(nameof(runspace));
+            }
+
+            PowerShell result = Create();
+
```

Comment:
nitpick: this new line can be removed.

Reply:
Agreed, both done. Thanks for the review!